### PR TITLE
`GOVERNANCE`  and `CONTRIBUTING`: single source of truth via references

### PR DESCRIPTION
**Pull request by Dominik-K**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
+Thank you for your interest in contributing to EVerest.
+If you just need help or have a question, refer to [SUPPORT.md](SUPPORT.md).
 
-* [How to Contribute a Bug Fix or Change](#How-to-Contribute-a-Bug-Fix-or-Change)
-* [Development Workflow](#Development-Workflow)
-* [Coding Style](#Coding-Style)
+This document explains our contribution process and procedures.
 
-For a description of the roles and responsibilities of the various members of the EVerest community, see the [governance policies], and for further details, see the project's [Technical Charter]. Briefly, Contributors are anyone who submits content to the project, Committers review and approve such submissions, and the Technical Steering Committee provides general project oversight.
+The EVerest community has various member roles: Contributor isyone who submits content to the project, Committers review and approve such submissions, and the Technical Steering Committee provides general project oversight.
```
Comment (hikinggrass) [inline CONTRIBUTING.md]: type: should be "Contributor is anyone..."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
-If you just need help or have a question, refer to [SUPPORT.md](SUPPORT.md).
+All contributors must comply with the project's [Technical Charter].
 
-## How to Contribute a Bug Fix or Change
+## Weekly tech sync meeting
+
+Every week, there's an online meeting open for everyone. Just come by and say hello 🙂.
 
-To contribute code to the project, first read over the [governance policies] page to understand the roles involved. 
+See the [EVerest calendar](https://lists.lfenergy.org/g/everest/calendar) for the dates. There you can also subscribe it.
```
Comment (hikinggrass) [inline CONTRIBUTING.md]: grammar: There you can also subscribe to this calendar.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Dominik-K) [reply CONTRIBUTING.md]: Yeah, thx. I think "everyone" works too, but sounds nice, IMO.

**Comment by krealyt** [comment]
@hikinggrass 
This one is now ready for a final review, I think. I did some changes due to another PR (https://github.com/EVerest/EVerest/pull/138) which updated similar topics in the CONTRIBUTING.md.

This one here is now mainly about "single source of truth" in GOVERNANCE.md.

**Review by Pietfried — APPROVED** [review]

**Comment by Pietfried** [comment]
lgmt, please resolve the conflicts before merging

**Pull request merged**